atlasdb-processors/src/main/java/com/palantir/processors/AutoDelegateProcessor.java

```diff
@@ -139,6 +141,20 @@ private static TypeElement validateAnnotatedElement(Element annotatedElement) th
         return (TypeElement) annotatedElement;
     }
 
+    private static void validateMethodsToBeAutoDelegated(Element annotatedElement, TypeToExtend typeToExtend)
+            throws ProcessingException {
+        Set<ExecutableElement> methodsToBeDelegated = typeToExtend.getMethods();
+
+        for (ExecutableElement methodElement : methodsToBeDelegated) {
+            if (methodElement.getModifiers().contains(Modifier.DEFAULT)
+                && methodElement.getAnnotation(DoDelegate.class) == null
```

Comment:
I haven't checked, but is there a neater method for doing `methodElement.getAnnotation(DoDelegate.class) == null`, like maybe `checkAnnotationPresent` or something like that?